<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRealisasi</class>
 <widget class="QWidget" name="FormRealisasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="kodeRealisasiLabel">
      <property name="text">
       <string>Kode Realisasi</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kodeRealisasiLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="kodeCadanganLabel">
      <property name="text">
       <string>Kode Cadangan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="kodeCadanganLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="dSBLabel">
      <property name="text">
       <string>DSB</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="dSBLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="iSBLabel">
      <property name="text">
       <string>ISB</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="iSBLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="tSBLabel">
      <property name="text">
       <string>TSB</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="tSBLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="pSBLabel">
      <property name="text">
       <string>PSB</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="pSBLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Form Realisasi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
